plugin-base/src/main/java/io/everytrade/server/parser/exchange/GbApiTransactionBean.java

```diff
@@ -107,16 +135,26 @@ public ImportedTransactionBean toImportedTransactionBean() {
             throw new DataValidationException(e.getMessage());
         }
 
-
-        return new ImportedTransactionBean(
-            uid,                             //uuid
-            timestamp,                       //executed
-            base,                            //base
-            quote,                           //quote
-            TransactionType.valueOf(action), //action
-            quantity,                        //base quantity
-            volume.divide(quantity, 10, RoundingMode.HALF_UP), //unit price
-            BigDecimal.ZERO                  //fee quote
+        return new TransactionCluster(
```

Comment:
should properly check fee currency and set ignored fee count for the cluster appropriately